GiniVision/Classes/Core/CapturedImagesStackView.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  CapturedImagesStackView.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 4/25/18.
+//
+
+import UIKit
+
+final class CapturedImagesStackView: UIView {
+    
+    enum State {
+        case filled(count: Int, lastImage: UIImage), empty
+    }
+    
+    let thumbnailSize: CGSize = {
+        if UIDevice.current.isIpad {
+            return CGSize(width: 40, height: 60)
+        } else {
+            return CGSize(width: 30, height: 45)
+        }
+    }()
+    var didTapImageStackButton: (() -> Void)?
+    fileprivate let stackCountCircleSize = CGSize(width: 25, height: 25)
+    fileprivate var imagesCount: Int = 0
+    
+    fileprivate lazy var thumbnailButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.layer.shadowColor = UIColor.black.cgColor
+        button.layer.shadowRadius = 1
+        button.layer.shadowOpacity = 0.5
+        button.layer.shadowOffset = CGSize(width: -2, height: 2)
+        button.addTarget(self, action: #selector(thumbnailButtonAction), for: .touchUpInside)
+        
+        return button
+    }()
+    
+    lazy var thumbnailStackBackgroundView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .lightGray
+        view.isHidden = true
+        return view
+    }()
+    
+    fileprivate lazy var stackIndicatorLabel: UILabel = {
+        var label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = Colors.Gini.blue
+        return label
+    }()
+    
+    fileprivate lazy var stackIndicatorCircleView: UIView = {
+        var view = UIView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.frame.size = self.stackCountCircleSize
+        view.backgroundColor = .white
+        view.layer.cornerRadius = self.stackCountCircleSize.width / 2
+        view.layer.shadowRadius = 1
+        view.layer.shadowOpacity = 0.3
+        view.layer.shadowOffset = CGSize(width: -1, height: 1)
+        return view
+    }()
+    
+    fileprivate lazy var capturedImagesStackSubtitleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = NSLocalizedString("ginivision.camera.capturedImagesStackLabel",
+                                       bundle: Bundle(for: GiniVision.self),
+                                       comment: "label shown below images stack")
+        label.textAlignment = .center
+        label.textColor = .white
+        label.font = label.font.withSize(12)
```

Comment:
Didn't you create an enum for common font sizes at one point?

Reply:
The customization is not part of this PR since it has not been defined yet. What you mean is a `customFont` property in the `GiniConfiguration`.
This will probably be customizable, but not decided yet.

Reply:
👍 Added [issue](https://github.com/gini/gini-vision-lib-ios/issues/209) to tackle this in a future PR.